<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data_add</class>
 <widget class="QWidget" name="data_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>file</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit_file">
              <property name="placeholderText">
               <string>File...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_file">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>List:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_list"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>first row:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_frow"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>num</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_num"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>serial</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_serial"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_read_file">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>ok</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_view">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>view</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget_data">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>25</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>25</number>
            </attribute>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>num</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>serial</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>0662058931</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>0123456789123</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Operator:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_operator"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_load">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>load</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_report">
           <property name="title">
            <string>report</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_report">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
